src/sql/azdata.proposed.d.ts

```diff
@@ -121,7 +121,8 @@ declare module 'azdata' {
 	 * Add OssRdbms for sqlops AzureResource.
 	 */
 	export enum AzureResource {
-		OssRdbms = 2
+		OssRdbms = 2,
```

Comment:
This is not really related to your PR and I may be missing context here, but why is this enum different from the one in interfaces? This seems to be missing `Sql` and `ResourceManagement`. @aaomidi any idea?

Reply:
Because this is the newly-added ones to the public api so are in the proposed typings file. The rest are in https://github.com/microsoft/azuredatastudio/blob/master/src/sql/azdata.d.ts#L2210

Reply:
Isn't proposed supposed to be a super set of the actual API? Or do we only add the new additions here with the actual API as a fallback?

Reply:
What do you mean fallback? Proposed is where new additions to the API go to be tested - things in proposed are able to have breaking changes made to them if we want to. This is the contract we've adopted from VS Code.

https://code.visualstudio.com/api/advanced-topics/using-proposed-api